### ISSUE TITLE
Acess denied / zstandart module

### ISSUE THREAD

**Issue by Yoko11111111** [Nuitka/Nuitka#1829]
* Please check whether the bug was already reported or fixed.

My Problem is if i try to open the exe it says acess denied maybe i dont have permissions something like that. Ive got an error:
Nuitka-Onefile:WARNING: Onefile mode cannot compress without 'zstandard' module installed.
I tried to install zstandart but: ERROR: No matching distribution found for zstandart. pls help

I tried this command: python -m nuitka --mingw64 --windows-icon-from-ico=ico.ico --onefile --windows-company-name=Zuna --windows-product-version=1.0 --plugin-enable=tk-inter main.py

* Nuitka version, full Python version, flavor, OS, etc. as output by *this*

1.1.2
Commercial: None
Python: 3.10.7 (tags/v3.10.7:6cc6b13, Sep  5 2022, 14:08:36) [MSC v.1933 64 bit (AMD64)]
Flavor: Unknown
Executable: C:\Users\jamu\AppData\Local\Programs\Python\Python310\python.exe
OS: Windows
Arch: x86_64
WindowsRelease: 10

* How did you install Nuitka and Python

 **normal pip / standart interpreter**
**pip install nuitka**

* The specific PyPI names and versions

2captcha-python | 1.1.2 | 1.1.2
-- | -- | --
Eel | 0.14.0 | 0.14.0
Faker | 15.0.0 | 15.0.0
Flask | 1.1.4 | 2.2.2
Jinja2 | 2.11.3 | 3.1.2
Js2Py | 0.71 | 0.71
MarkupSafe | 2.1.1 | 2.1.1
MouseInfo | 0.1.3 | 0.1.3
Nuitka | 1.1.2 | 1.1.2
Pillow | 9.2.0 | 9.2.0
PyAutoGUI | 0.9.53 | 0.9.53
PyGetWindow | 0.0.9 | 0.0.9
PyMsgBox | 1.0.9 | 1.0.9
PyPrind | 2.11.3 | 2.11.3
PyRect | 0.2.0 | 0.2.0
PyScreeze | 0.1.28 | 0.1.28
PySocks | 1.7.1 | 1.7.1
PyYAML | 6.0 | 6.0
SpeechRecognition | 3.8.1 | 3.8.1
VaLID | 1.0.8 | 1.0.8
Werkzeug | 1.0.1 | 2.2.2
actions | 1.0.3 | 1.0.3
aiohttp | 3.8.1 | 3.8.3
aiosignal | 1.2.0 | 1.2.0
altgraph | 0.17.3 | 0.17.3
anyio | 3.6.1 | 3.6.1
appdirs | 1.4.4 | 1.4.4
astunparse | 1.6.3 | 1.6.3
async-generator | 1.10 |  
async-timeout | 4.0.2 | 4.0.2
attrs | 22.1.0 | 22.1.0
auto-py-to-exe | 2.23.1 | 2.23.1
b | 1.0.0 | 1.0.0
barrier | 0.2.2 | 0.2.2
bcolors | 1.0.4 | 1.0.4
beautifulsoup4 | 4.11.1 | 4.11.1
bottle | 0.12.23 | 0.12.23
bottle-websocket | 0.2.9 | 0.2.9
bs4 | 0.0.1 | 0.0.1
bunch | 1.0.1 | 1.0.1
buttons | 0.1.9 | 0.1.9
certifi | 2022.6.15 | 2022.9.24
cffi | 1.15.1 | 1.15.1
charset-normalizer | 2.1.1 | 2.1.1
click | 7.1.2 | 8.1.3
codeclose | 1.0 |  
colorama | 0.4.4 | 0.4.5
configobj | 5.0.6 | 5.0.6
configure | 0.5 | 0.5
contourpy | 1.0.5 | 1.0.5
cryptography | 38.0.1 | 38.0.1
curlify | 2.2.1 | 2.2.1
cursor | 1.3.5 | 1.3.5
cycler | 0.11.0 | 0.11.0
decorator | 5.1.1 | 5.1.1
discord.py | 2.0.1 | 2.0.1
distlib | 0.3.6 | 0.3.6
docopt | 0.6.2 | 0.6.2
driver | 0.1.0 | 0.1.0
fake-headers | 1.0.2 | 1.0.2
fake-useragent | 0.1.11 | 0.1.11
filelock | 3.8.0 | 3.8.0
flask-oidc | 1.4.0 | 1.4.0
fonttools | 4.37.4 | 4.37.4
frozenlist | 1.3.1 | 1.3.1
future | 0.18.2 | 0.18.2
gevent | 21.12.0 | 21.12.0
gevent-websocket | 0.10.1 | 0.10.1
greenlet | 1.1.3 | 1.1.3
gunicorn | 19.10.0 | 20.1.0
h11 | 0.12.0 | 0.14.0
html5lib | 1.1 | 1.1
httmock | 1.4.0 | 1.4.0
http-request-randomizer | 1.3.2 | 1.3.2
httpcore | 0.13.7 | 0.15.0
httplib2 | 0.20.4 | 0.20.4
httpx | 0.19.0 | 0.23.0
idna | 3.3 | 3.4
iso-639 | 0.4.5 | 0.4.5
iso8601 | 1.0.2 | 1.1.0
isodate | 0.6.1 | 0.6.1
itsdangerous | 1.1.0 | 2.1.2
joblib | 1.2.0 | 1.2.0
kiwisolver | 1.4.4 | 1.4.4
licensing | 0.37 | 0.37
lxml | 4.9.1 | 4.9.1
matplotlib | 3.6.0 | 3.6.0
multidict | 6.0.2 | 6.0.2
names | 0.3.0 | 0.3.0
numpy | 1.23.3 | 1.23.3
oauth2client | 4.1.3 | 4.1.3
ordered-set | 4.1.0 | 4.1.0
orq | 0.0.1 | 0.0.1
outcome | 1.2.0 | 1.2.0
packaging | 21.3 | 21.3
pandas | 1.5.0 | 1.5.0
pefile | 2022.5.30 | 2022.5.30
pip | 22.2.2 | 22.2.2
pipenv | 2022.9.21 | 2022.10.4
pipwin | 0.5.2 | 0.5.2
platformdirs | 2.5.2 | 2.5.2
playwright | 1.26.1 | 1.26.1
poetry-version | 0.1.5 | 0.2.0
psutil | 5.9.2 | 5.9.2
pyOpenSSL | 22.0.0 | 22.1.0
pySmartDL | 1.3.4 | 1.3.4
pyasn1 | 0.4.8 | 0.4.8
pyasn1-modules | 0.2.8 | 0.2.8
pycountry | 22.3.5 | 22.3.5
pycparser | 2.21 | 2.21
pycryptodome | 3.15.0 | 3.15.0
pycryptodomex | 3.15.0 | 3.15.0
pyee | 8.1.0 | 9.0.4
pygame | 2.1.2 | 2.1.2
pyinstaller | 5.4.1 | 5.4.1
pyinstaller-hooks-contrib | 2022.10 | 2022.10
pyjsparser | 2.7.1 | 2.7.1
pyparsing | 3.0.9 | 3.0.9
pyperclip | 1.8.2 | 1.8.2
pystyle | 2.0 | 2.0
python-dateutil | 2.8.2 | 2.8.2
python-dotenv | 0.21.0 | 0.21.0
pytweening | 1.0.4 | 1.0.4
pytz | 2022.2.1 | 2022.4
pytz-deprecation-shim | 0.1.0.post0 | 0.1.0.post0
pywin32 | 304 | 304
pywin32-ctypes | 0.2.0 | 0.2.0
requests | 2.28.1 | 2.28.1
rfc3986 | 1.5.0 | 2.0.0
rsa | 4.9 | 4.9
scikit-learn | 1.1.2 | 1.1.2
scipy | 1.9.1 | 1.9.1
selenium | 4.4.3 | 4.5.0
self | 2020.12.3 | 2020.12.3
setuptools | 63.2.0 | 65.4.1
six | 1.16.0 | 1.16.0
sklearn | 0.0 | 0.0
sniffio | 1.3.0 | 1.3.0
sortedcontainers | 2.4.0 | 2.4.0
soupsieve | 2.3.2.post1 | 2.3.2.post1
streamlink | 5.0.1 | 5.0.1
sudo | 1.0.0 | 1.0.0
termcolor | 2.0.1 | 2.0.1
threadpoolctl | 3.1.0 | 3.1.0
tomlkit | 0.5.11 | 0.11.5
tqdm | 4.64.1 | 4.64.1
trio | 0.21.0 | 0.22.0
trio-websocket | 0.9.2 | 0.9.2
twitchAPI | 2.5.7.1 | 2.5.7.1
twitchio | 2.4.0 | 2.4.0
typing-extensions | 4.3.0 | 4.3.0
tzdata | 2022.2 | 2022.4
tzlocal | 4.2 | 4.2
ui | 0.1.4 | 0.1.4
undetected-chromedriver | 3.1.5.post4 | 3.1.5.post4
urllib3 | 1.26.12 | 1.26.12
user-agent | 0.1.10 | 0.1.10
useragent | 0.1.1 | 0.1.1
virtualenv | 20.16.5 | 20.16.5
virtualenv-clone | 0.5.7 | 0.5.7
webdriver-manager | 3.8.3 | 3.8.3
webencodings | 0.5.1 | 0.5.1
websocket-client | 1.4.1 | 1.4.1
websockets | 10.1 | 10.3
wheel | 0.37.1 | 0.37.1
whichcraft | 0.6.1 | 0.6.1
wsproto | 1.2.0 | 1.2.0
yarl | 1.8.1 | 1.8.1
zope.event | 4.5.0 | 4.5.0
zope.interface | 5.4.0 | 5.4.0

* Also supply a [Short, Self Contained, Correct, Example](https://sscce.org/)

idk what to insert

**Comment by kayhayen** [comment]
The module is named `zstandard`, I generally recommend using copy&paste for names.

I guess, you would prefer from Nuitka asking if it may install it a lot, unfortunately there are way too many package managers to do that sanely.

**Comment by Yoko11111111** [comment]
> 

Yeah i see i got the module but i still dont have acess

**Comment by kayhayen** [comment]
Posting incomplete information below an issue is getting us nowhere.
